_tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore.md

```diff
@@ -296,6 +296,55 @@ Use the following steps to register a snapshot repository backed by an Azure sto
    }
    ```
 
+### General set up to use Microsoft Azure Blob Storage
+
+To utilize Microsoft Azure Blob Storage as a snapshot repository, follow the steps below:
+1. Install the `repository-azure` plugin on all nodes with the following command:
+
+   ```bash
+   ./bin/opensearch-plugin install repository-azure
+   ```
+
+1. Once the repository-azure plugin is installed, define your Azure Blob Storage settings before initializing the node. Start by defining your Azure Storage account name using the following secure setting:
+
+   ```bash
+   ./bin/opensearch-keystore add azure.client.default.account
+   ```
+
+1. choose one of the following options to set up your authentication credentials for Azure Blob Storage:
+
+   #### Using Azure storage account key
```

Comment:
Let's fix the spacing here since this heading won't format under an ordered step.

```suggestion
#### Using Azure storage account key
```